hazelcast/src/main/java/com/hazelcast/nio/tcp/iobalancer/IOBalancer.java

```diff
@@ -157,39 +158,43 @@ private MigrationStrategy createMigrationStrategy() {
             strategy = new MonkeyMigrationStrategy();
         } else {
             log.finest("Using normal IO Balancer Strategy.");
-            strategy =  new EventCountBasicMigrationStrategy();
+            strategy = new EventCountBasicMigrationStrategy();
         }
         return strategy;
     }
 
     private boolean isEnabled(InSelectorImpl[] inSelectors, OutSelectorImpl[] outSelectors) {
-        if (migrationIntervalSeconds < 0) {
-            if (log.isFinestEnabled()) {
-                log.finest("I/O Balancer is disabled as the '"
-                        + GroupProperties.PROP_PERFORMANCE_MONITORING_ENABLED + "' property is set to "
-                        + migrationIntervalSeconds + ". Set the property to a positive value to enable I/O Balancer.");
-            }
+        if (balancerIntervalSeconds < 0) {
+            log.warning("I/O Balancer is disabled as the '"
+                    + PROP_IO_BALANCER_INTERVAL_SECONDS + "' property is set to "
+                    + balancerIntervalSeconds + ". Set the property to a positive value to enable the I/O Balancer.");
             return false;
         }
+
         if (inSelectors.length == 1 && outSelectors.length == 1) {
             log.finest("I/O Balancer is disabled as there is only a single a pair of I/O threads. Use the '"
-                    + GroupProperties.PROP_IO_THREAD_COUNT + "' property to increase number of I/O Threads.");
+                    + PROP_IO_THREAD_COUNT + "' property to increase number of I/O Threads.");
             return false;
         }
+
+        if (log.isFinestEnabled()) {
+            log.finest("I/O Balancer is enabled. Scanning every " + balancerIntervalSeconds + " seconds for imbalances.");
+        }
+
         return true;
     }
 
     private void tryMigrate(LoadImbalance loadImbalance) {
         MigratableHandler handler = strategy.findHandlerToMigrate(loadImbalance);
         if (handler == null) {
-            log.finest("There had been I/O imbalance detected, but no suitable migration candidate was found.");
+            log.finest("I/O imbalance are detected, but no suitable migration candidate is found.");
             return;
         }
 
         IOSelector destinationSelector = loadImbalance.destinationSelector;
         if (log.isFinestEnabled()) {
             IOSelector sourceSelector = loadImbalance.sourceSelector;
-            log.finest("Scheduling a migration of a handler " + handler
+            log.finest("Scheduling a migration of handler " + handler
                     + " from a selector thread " + sourceSelector + " to " + destinationSelector);
```

Comment:
Would remove the "a" in "from a selector thread" as well. And maybe "the migration" instead of "a migration", but I'm not 100% sure about that.

```
log.finest("Scheduling the migration of handler " + handler
        + " from selector thread " + sourceSelector + " to " + destinationSelector);
```